server/api.js

```diff
@@ -196,6 +230,42 @@ router.put("/graduates/:id", function (req, res) {
     ],
     (error, result) => {
       if (result) {
+        Connection.query(
+          `select id from graduates where github_id=$1`,
+          [github_id],
+          (error, result) => {
+            if (result) {
+              let graduate_id = result.rows[0].id;
+              Connection.query(
+                `delete from graduate_skill where graduate_id =$1`,
+                [graduate_id],
+                (error, result) => {
+                  console.log(skills);
+                  skills.map((s) => {
+                    Connection.query(
+                      `select id from skills where skill_name =$1`,
+                      [s],
+                      (error, result) => {
+                        if (result) {
+                          let skill_id = result.rows[0].id;
+                          Connection.query(
+                            `insert into graduate_skill values ($1,$2)`,
+                            [graduate_id, skill_id],
+                            (error, result) => {
+                              if (result) {
+                                console.log("post request for skills ", result);
```

Comment:
meaningful log would be better, for example logging the result or even the parameters